### PR TITLE
[ONNX] Add support for torch.bernoulli() export

### PR DESCRIPTION
**Pull request by hwangdeyu**
Support export HuggingFace - Training DeBERTa model.

### CODE REVIEW

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -2371,6 +2371,19 @@ def rrelu(g, input, lower, upper, training, generator):
     return g.op('PRelu', input, p)
 
 
+def bernoulli(g, input, generator=None, out=None):
+    if generator is not None and not sym_help._is_none(generator):
+        _unimplemented("Bernoulli", "generator is not supported for multinomial")
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset9.py:2376]:
```suggestion
        _unimplemented("Bernoulli", "generator is not supported for Bernoulli")
```

**Review by shubhambhokare1 — APPROVED** [review]
LGTM

File: torch/onnx/symbolic_opset9.py [hunk 2]
```diff
@@ -2371,6 +2371,19 @@ def rrelu(g, input, lower, upper, training, generator):
...
 
+def bernoulli(g, input, generator=None, out=None):
+    if generator is not None and not sym_help._is_none(generator):
+        _unimplemented("Bernoulli", "generator is not supported for bernoulli")
+
+    self = input if out is None else out
+    dtype = sym_help._try_get_scalar_type(self)
+    if dtype is None:
+        dtype = "Float"
+    p = g.op('RandomUniformLike', input, high_f=1.0, low_f=0.0)
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset9.py:2382]: dtype is not provided for `RandomUniformLike`, thus type of `p` defaults to float. As a result, below `Less` operator will have dtype mismatch if `input` has a different type.

**Review by BowenBao — CHANGES_REQUESTED** [review]

File: torch/onnx/symbolic_opset9.py (hunk 2 above)
Comment (hwangdeyu) [reply torch/onnx/symbolic_opset9.py]: Thanks, fix the RandomUniformLike result dtype.

**Review by neginraoof — APPROVED** [review]
LGTM. Thanks.

File: .gitmodules [hunk 3]
```diff
@@ -74,6 +74,10 @@
     ignore = dirty
     path = third_party/onnx
     url = https://github.com/onnx/onnx.git
+[submodule "third_party/optimizer"]
```
Comment (hwangdeyu) [inline .gitmodules:77]: those code should not be here from https://github.com/pytorch/pytorch/pull/55889

**Review by BowenBao — APPROVED** [review]

**Pull request merged**